internetarchive/cli/ia_upload.py

```diff
@@ -85,8 +86,23 @@
 )
 
 
+def _check_files(files):
```

Comment:
Should have a better name e. g. `_check_files_exist` or something else which makes it clear that it merely checks whether a path leads to an actual file.

Reply:
Ye I saw them now. Thank you for your answer. Another problem that I noticed is that `files` is an immutable data type, so altering it might not be a good idea.